src/Effects/Panning.icl

```diff
@@ -57,67 +44,68 @@ sineList = [x\\x<-:sineTable]
 
 
 ampPanMonoCircular :: [Real] Real Direction ->([Real],[Real])
-ampPanMonoCircular wave panVal dir
+ampPanMonoCircular wave pan_val dir
 | dir == Left = ([x*loudness\\x<-wave],[x*(1.0-loudness)\\x<-wave])
 =([x*(1.0-loudness)\\x<-wave],[x*loudness\\x<-wave])
 where
 	PI=3.14
-	dist=sqrt(2.0*115.5625 - 2.0*115.5625*cos((1.0-panVal) * PI/2.0))
+	dist=sqrt(2.0*115.5625 - 2.0*115.5625*cos((1.0-pan_val) * PI/2.0))
```

Comment:
make a constant for 115.5625
does not make any sense when I look at it.

Reply:
I'll check it now thank you!!!